README.md

```diff
@@ -4,33 +4,66 @@ gdk-ocean is a Liquid wallet running on a gRPC server. It implements the [ocean
 
 # Usage
 
-## Server
+## Run gdk-ocean locally
 
-The current version of gdk-ocean uses Python 3.9, see `requirements.txt` for dependencies.
+The current version of gdk-ocean uses Python 3.9
 
 Run the server in any Python 3.9 environmnent:
 ```
 make run
 ```
+By default, the server is running on port 50051.
 
-## CLI
-
-You can communicate with gdk-ocean using the `cli.py` script. 
-
-### Development
+__the first time you run the program, you have to set up the virtual env with all dependencies.__
 
-Create virtual env (need Python 3.9 installed)
+Create virtual env (it needs Python 3.9 installed)
 ```
 make env
 ```
 
-Then, activate the env and install the dependencies.
+Then, activate the env and install the `requirements.txt` deps.
 ```
 source venv/bin/activate
+make stubs
 make deps
 ```
 
-Run the gdk-ocean instance
+At any moment, you can leave the venv with `deactivate`.
+
+## Run gdk-ocean in a Docker container
+
+Build the image:
+```bash
+docker build -t gdk-ocean . # -t is the image name
 ```
-make run
-```
\ No newline at end of file
+
+Run a container:
+```bash
+docker run gdk-ocean 
+```
+> You may need to set `--net=host` option on linux OS if you plan to request the server via the `localhost` address.
```

Comment:
Remove this, the problem is that the service runs on hardcoded host `localhost`, while it should run on `0.0.0.0` when dockerized.
This is solvable by defining a `--address` flag instead of `--port` one.